optaplanner-examples/src/test/java/org/optaplanner/examples/nurserostering/score/NurseRosteringConstraintProviderTest.java

```diff
@@ -403,27 +402,21 @@ void minimumAndMaximumNumberOfAssignments(
         ShiftAssignment shift4 = getShiftAssignment(2, employee);
         ShiftAssignment shift5 = getShiftAssignment(2, employee);
 
-        constraintVerifier.verifyThat(NurseRosteringConstraintProvider::minimumAndMaximumNumberOfAssignments)
-                .given(contract.getContractLineList().get(0),
-                        employee,
-                        shift1, shift2, shift3, shift4, shift5)
+        SingleConstraintVerification<NurseRoster> minimumAndMaximumNumberOfAssignmentsConstraint =
+                constraintVerifier.verifyThat(NurseRosteringConstraintProvider::minimumAndMaximumNumberOfAssignments);
+
+        minimumAndMaximumNumberOfAssignmentsConstraint
+                .given(contract.getContractLineList().get(0), employee, shift1, shift2, shift3, shift4, shift5)
```

Comment:
`contract.getContractLineList().get(0)` repeats multiple times; maybe we could extract a method for that.